src/rgw/rgw_lc.cc

```diff
@@ -1702,7 +1702,7 @@ int RGWLC::bucket_lc_process(string& shard_id, LCWorker* worker,
       << dendl;
     int ret = op_rule.process(o, wk->dpp, wq);
     if (ret < 0) {
-      ldpp_dout(wk->get_lc(), 20)
+      ldpp_dout(wk->get_lc(), 0)
```

Comment:
I'm sort of suspicious about level 0 logging--this isn't a system error;  if we are going to make this that prominent, is this message telling enough to identify and help resolve an issue if there is one?  I think the lack of that is why it's currently at 20

Reply:
Ahh, yes. That makes sense. Reverted back to its original value (20) in the new evidence.